partner_auto_match/models/res_partner_match.py

```diff
@@ -107,7 +107,7 @@ def update_partner(self, partner, vals, async_mode=True, delay=1):
     @api.model
     def _preprocess_vals(self, vals):
         """Transform, if needed and before matching, the infos received"""
-        pass
+        vals["name"] = vals["name"].strip(" -")
```

Comment:
Maybe check that "name" is present, in case other rules don't receive "name" in the vals.
```suggestion
        if "name" in vals:
            vals["name"] = vals["name"].strip(" -")
```